adjust_gamma.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
```

Comment:
Please remove the .pyc files as they are not code and runtime outputs. Avoid commiting binary files unless used for testing.